test_series.py

```diff
@@ -1 +1,113 @@
+"""Tests for series function."""
+import pytest
 
+
+FIBONACCI_NUMBERS = [
+    [0, 0],
+    [1, 1],
+    [2, 1],
+    [3, 2],
+    [4, 3],
+    [5, 5],
+    [6, 8],
+    [7, 13],
+    [8, 21],
+    [9, 34],
+    [10, 55],
+    [30, 832040],
+]
+
+
+LUCAS_NUMBERS = [
+    [0, 2],
+    [1, 1],
+    [2, 3],
+    [3, 4],
+    [4, 7],
+    [5, 11],
+    [6, 18],
+    [7, 29],
+    [8, 47],
+    [9, 76],
+    [10, 123],
+    [30, 1860498],
+]
+
+
+WILSON_NUMBERS = [
+    [6, 5, 2, 41],
+    [4, 6, 9, 39],
+    [5, 4, 2, 22],
+    [7, 3, 10, 154],
+]
+
+
+@pytest.mark.parametrize("n, result", FIBONACCI_NUMBERS)
+def test_fibonacci(n, result):
+    """Test fibonacci for some value of n."""
+    from series import fibonacci
+    assert fibonacci(n) == result
+
+
+def test_fibonacci_exists():
+    """Test whether fibonacci function exists."""
+    import series
+    assert series.fibonacci
+
+
+def test_fib_0():
```

Comment:
Since this input/result combo is already included in your `FIBONACCI_NUMBERS` table, this test function is redundant and un-needed.

Remember, don't repeat yourself. No un-needed code!